src/dodal/devices/zebra_controlled_shutter.py

```diff
@@ -0,0 +1,31 @@
+from enum import Enum
+
+from bluesky.protocols import Movable
+from ophyd_async.core import AsyncStatus, StandardReadable, wait_for_value
+from ophyd_async.epics.signal import epics_signal_r, epics_signal_w
+
+
+class ZebraShutterState(str, Enum):
+    CLOSE = "Close"
+    OPEN = "Open"
+
+
+class ZebraShutter(StandardReadable, Movable):
+    def __init__(self, prefix: str, name: str):
+        self.position_set = epics_signal_w(
+            write_pv=prefix + "CTRL2",
+            datatype=ZebraShutterState,
+        )
+        with self.add_children_as_readables():
+            self.position_readback = epics_signal_r(
+                read_pv=prefix + "STA",
+                datatype=ZebraShutterState,
+            )
```

Comment:
nit: these signals used to be called `pos` and `pos_rbv`

If Hyperion expects them to be called something in particular and ever reads rather than just setting this is a Must not a Nit.

Reply:
@DominicOram how do you stand about this? I'm all for readable variable names

Reply:
I'm also all for readable names, and Since the device name is changing it will have to be updated in Hyperion anyway. I would like it if you would update the Hyperion PR to match this one. Also may I suggest `position_setpoint` instead?

Reply:
added `position_setpoint`